### PR TITLE
Add metadata to skip failing images

### PR DESCRIPTION
**Pull request by RDIL**
@TravisEz13 please merge this without the 24 hour period, I really want to see if this fixes it.

### CODE REVIEW

File: tools/buildHelper/buildHelper.psm1 [hunk 1]
```diff
@@ -92,7 +92,7 @@ function Get-ImageList
 
     if ($Channel -in 'community-stable', 'all')
     {
-        Get-ChildItem -Path $communityStablePath -Directory | Select-Object -ExpandProperty Name | Where-Object { $dockerFileNames -notcontains "clearlinux" } | Write-Output
+        Get-ChildItem -Path $communityStablePath -Directory | Select-Object -ExpandProperty Name | Where-Object { $dockerFileNames -notcontains "clearlinux" } | Where-Object { $dockerFileNames -notcontains "parrot" } | Where-Object { $dockerFileNames -contains "kali" } | Write-Output
```
Comment (TravisEz13) [inline tools/buildHelper/buildHelper.psm1]: I was fine with this when it was one, but let's make this a property in metadata.json.

File: release/community-stable/clearlinux/meta.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "IsLinux" : true,
     "PackageFormat": "powershell-${PS_VERSION}-linux-x64.tar.gz",
     "SkipGssNtlmSspTests": true,
-    "SkipWebCmdletTests": false,
+    "IsBroken": true,
```
Comment (TravisEz13) [inline release/community-stable/clearlinux/meta.json]:
```suggestion
    "SkipWebCmdletTests": false,
    "IsBroken": true,
```

Comment (RDIL) [reply release/community-stable/clearlinux/meta.json]: The default is false so this isn't needed

**Comment by RDIL** [comment]
This is the part where I ask for help with the syntax error

**Comment by TravisEz13** [comment]
The json gets converted into a type.  You did not add the property into the type... I didn't look for any further problems... feel free to ping me again.

**Comment by RDIL** [comment]
Anything blocking it?

**Pull request merged**